testcafe/service.js

```diff
@@ -61,41 +57,40 @@ test('Confirm Page Loads with Information', async t => {
     .ok();
 });
 
-test('Confirm Service Schedule Day Can Be Added', async t => {
+test.skip('Confirm Service Schedule Day Can Be Added', async t => {
   // Wait for page to load before counting services by using hover action.
-  await t.hover(servicePage.editButton);
+  await t.hover(servicePage.schedule);
 
   // Count the number of schedule days
   const originalScheduleDayCount = await servicePage.schedule.with({ boundTestRun: t }).count;
 
-  await modifySheduleTime(t);
+  await t.navigateTo(servicePage.url(serviceId));
+
+  await modifySheduleTime(t, 'add');
 
   await t
-    .navigateTo(servicePage.url(serviceId))
-    .hover(servicePage.editButton)
+    .navigateTo(editResourcePage.url(1))
```

Comment:
I finally got a chance to take a look at the error you were running into, @ashleyabrooks, and I think this is the bug. Previously we were looking at the (non-edit) Service Page to get the count of schedule days, but this line changed it to go to the Edit Page instead.

I'm not sure why the `.hover(servicePage.editButton)` was there (might have been to force a page scroll), but it wasn't actually clicking on the button, so we should still be on the Service Page after this action.

Let me submit a PR to fix this and unskip the tests.